.changeset/lovely-colts-fail.md

```diff
@@ -0,0 +1,5 @@
+---
+'@chainlink/ion.au-adapter': major
+---
+
+rate limit
```

Comment:
This change is probably more of a `patch` and could you expand on the message since this gets written to the CHANGELOG. Something like
```suggestion
'@chainlink/ion.au-adapter': patch
---

Reduced default rate limit to 5 requests per minute
```

Reply:
Hi @austpryb, I noticed this got marked as resolved but the changeset still looks to have the major version and old comment.

Reply:
Hi @austpryb, just wanted to bump this so I can get it merged and part of the next release